src/chain-data/context.tsx

```diff
@@ -14,6 +14,7 @@ const ChainDataContextProvider: React.FC = ({ children }) => {
 
         if (process.env.NODE_ENV === 'development') {
           console.group(reason);
+          // eslint-disable-next-line no-console
```

Comment:
Can't these `eslint-disable-next-line` statements be removed since `console.info` been added as an "allow" exception?